include/usyscalls.h

```diff
@@ -0,0 +1,10 @@
+#ifndef _USYSCALLS_H_
+#define _USYSCALLS_H_
+
+// Declarations for userland end of the system calls. Implemented in
+// usyscalls.s.
+
+extern void sys_puts(char const* msg);
```

Comment:
sys_puts() - probably it should be called just puts().
Otherwise there is discrepancy in naming between sys_puts() and getpid(), right now.

Reply:
I guess it should probably end up being called `write`... I just didn't want to clutter the PR with noise, I'll do it separately.